kolibri/plugins/device/assets/src/views/RearrangeChannelsPage.vue

```diff
@@ -36,8 +36,8 @@
                 >
                   <DragSortWidget
                     class="sort-widget"
-                    :moveUpText="$tr('upLabel', { name: channel.name })"
-                    :moveDownText="$tr('downLabel', { name: channel.name })"
+                    :moveUpText="moveUplabel(channel.name)"
```

Comment:
For this case you could do `:moveUpText="() => $tr('moveUpLabel', { name: channel.name })"` to avoid adding the computed properties.